src/Auth0.php

```diff
@@ -291,6 +294,37 @@ public function renew(
         }
     }
 
+    /**
+     * Return an object representing the current session state (including id token, access token, access token expiration, refresh token and user data) without triggering an authorization flow. Returns null when session data is not available.
+     */
+    public function getState(): ?object
```

Comment:
I fear "state" could be confused with the one we send over for authentication (i.e. state & nonce). Might be better to use a different name. In Android and Swift we call them [Credentials](https://github.com/auth0/Auth0.swift/blob/master/Auth0/Credentials.swift) for example

Reply:
That's a good call out, I think that offers better clarity too. I'll update the method name.